docs/interpreter/hive.md

```diff
@@ -161,3 +171,9 @@ GROUP BY ${group_by=product_id,product_id|product_name|customer_id|store_id}
 ORDER BY count ${order=DESC,DESC|ASC}
 LIMIT ${limit=10};
 ```
+
+### Impersonation
+When Zeppelin server is running with authentication enabled, then this interpreter utilizes Hive's user proxy feature i.e. sends extra parameter for creating and running a session ("hive.server2.proxy.user=": "${loggedInUser}"). This is particularly useful when multi users are sharing a Notebook server.
```

Comment:
Yes, that could happen, but it was never documented, hence thought to document it here.